apps/web/pages/settings/my-account/profile.tsx

```diff
@@ -439,10 +448,7 @@ const ProfileView = () => {
             <Button
               color="primary"
               onClick={() => {
-                updateProfileMutation.mutate({
```

Comment:
It didn't make much sense to have this being part of updateProfile. updateProfile is doing too much already.